src/components/cc-token-api-creation-form/cc-token-api-creation-form.js

```diff
@@ -0,0 +1,893 @@
+                required
+                .errorMessage=${mfaErrorMessage}
+                .value="${mfaCode}"
+                @cc-input-text:input=${this._onInput}
+              ></cc-input-text>
+            `
+          : ''}
+
+        <div class="form__actions">
+          <div class="form__actions__link-container">
+            ${isWaiting
+              ? html`
+                  <div class="form__actions__link-container__link">
+                    <cc-icon .icon=${iconGoBack}></cc-icon>
+                    <span>${i18n('cc-token-api-creation-form.validation-step.form.link.back-to-configuration')}</span>
+                  </div>
+                `
+              : ''}
+            ${!isWaiting
+              ? html`
+                  <a class="form__actions__link-container__link" @click="${this._onConfigLinkClick}" href="#">
+                    <cc-icon .icon=${iconGoBack}></cc-icon>
+                    <span>${i18n('cc-token-api-creation-form.validation-step.form.link.back-to-configuration')}</span>
+                  </a>
+                `
+              : ''}
+          </div>
+          <cc-button class="form__actions__submit-button" primary type="submit" ?waiting=${isWaiting}>
+            ${i18n('cc-token-api-creation-form.validation-step.form.submit-button.label')}
+          </cc-button>
+        </div>
+      </form>
+    `;
+  }
+
+  /** @param {string} token */
+  _renderCopyStep(token) {
+    return html`
+      <div class="copy-step-wrapper">
+        <cc-notice intent="warning" .message=${i18n('cc-token-api-creation-form.copy-step.notice.message')}></cc-notice>
+        <cc-input-text
+          label="${i18n('cc-token-api-creation-form.copy-step.form.token.label')}"
+          name="token"
+          readonly
+          secret
+          clipboard
+          value=${token}
+        ></cc-input-text>
+        <a class="token-list-link-cta" href="${this.apiTokenListHref}">
+          <span>${i18n('cc-token-api-creation-form.copy-step.link.back-to-list')}</span>
+        </a>
+      </div>
+    `;
+  }
+
+  static get styles() {
+    return [
+      linkStyles,
+      cliCommandsStyles,
+      css`
+        :host {
+          display: block;
+
+          --transition-duration: 300ms;
+        }
+
+        @media (prefers-reduced-motion) {
+          :host {
+            --transition-duration: 0ms;
+          }
+        }
+
+        /* FIXME: not great when viewport is reduced + should be handled by the cc-block component itself */
+        cc-block {
+          padding-top: 2em;
+        }
+
+        /* FIXME: not great when viewport is reduced + should be handled by the cc-block component itself */
+        cc-block > [slot='content'] {
+          padding-bottom: 2em;
+          padding-inline: 3em;
+        }
+
+        /* FIXME: not great when viewport is reduced + should be handled by the cc-block component itself */
+        [slot='header-title'] {
+          padding-inline: 1.5em;
+        }
```

Comment:
I could make this better with `container-queries` but #1387 blocks this, I don't really want to add the resize controller just for that so I suggest we merge as is and I tackle the `container-queries` issue right after?